AppWidget/glance-widget/src/main/java/com/example/android/glancewidget/FirstGlanceWidget.kt

```diff
@@ -47,26 +48,30 @@ class FirstGlanceWidget : GlanceAppWidget() {
         Column(
             modifier = GlanceModifier
                 .fillMaxSize()
-                .background(day = Color.White, night = Color.LightGray)
+                .background(day = Color.White, night = Color.Black)
```

Comment:
(For another commit) While we don't provide a GlanceTheme maybe we can at least showcase how to use resources to get the dynamic color here too?

Basically creating a color resource that is set for values/value-night and values-v31/values-v31-night. 

We cannot use attrs here, but maybe you can check the M3 library to use the same colors.

For example, for the v31 background they use 
```
<color name="gm3_ref_palette_dynamic_neutral99">@android:color/system_neutral1_10</color>
```

Reply:
I would agree. We might want to define clearly what a theme might look like though. Taking inspiration from Compose UI's themes.

Reply:
I might took Marcel's intention wrong, but are you talking about probably gm3 library instead of Material 3 library?
I couldn't find Material3 library does such a thing.